drivers/media/i2c/imx258.c

```diff
@@ -1782,6 +1817,19 @@ static int imx258_probe(struct i2c_client *client)
 
 	imx258->csi2_flags = ep.bus.mipi_csi2.flags;
 
+#ifdef CONFIG_ACPI
```

Comment:
Is this still necessary? i2c.h already provides a null (and NULL-returning) implementation of `i2c_of_match_device` for the case that `CONFIG_OF` is not enabled.

Reply:
Fixed. I'd initially used of_match_device, but then swapped.